content/zh-cn/docs/reference/kubernetes-api/workload-resources/job-v1.md

```diff
@@ -0,0 +1,1666 @@
+
+- **manualSelector** (boolean)
+
+  manualSelector controls generation of pod labels and pod selectors. Leave `manualSelector` unset unless you are certain what you are doing. When false or unset, the system pick labels unique to this job and appends those labels to the pod template.  When true, the user is responsible for picking unique labels and specifying the selector.  Failure to pick a unique label may cause this and other jobs to not function correctly.  However, You may see `manualSelector=true` in jobs that were created with the old `extensions/v1beta1` API. More info: https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#specifying-your-own-pod-selector
+-->
+- **selector** (<a href="{{< ref "../common-definitions/label-selector#LabelSelector" >}}">LabelSelector</a>)
+
+  对应与 Pod 计数匹配的 Pod 的标签查询。通常，系统会为你设置此字段。更多信息：
+  https://kubernetes.io/zh-cn/docs/concepts/overview/working-with-objects/labels/#label-selectors
+
+- **manualSelector** (boolean)
+
+  manualSelector 控制 Pod 标签和 Pod 选择器的生成。除非你确定你在做什么，否则不要设置 `manualSelector`。
+  当此字段为 false 或未设置时，系统会选择此 Pod 唯一的标签并将这些标签附加到 Pod 模板。
+  当此字段为 true 时，用户负责选择唯一标签并指定选择器。
+  未能选择唯一标签可能会导致此作业和其他作业无法正常运行。
+  但是，你可能会在使用旧的 `extensions/v1beta1` API 创建的作业中看到 `manualSelector=true`。
+  更多信息： https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#specifying-your-own-pod-selector
+
+## JobStatus {#JobStatus}
+
+
+<!--
+JobStatus represents the current state of a Job.
+-->
+JobStatus 表示 Job 的当前状态。
+
+<hr>
+
+<!--
+- **startTime** (Time)
+
+  Represents time when the job controller started processing a job. When a Job is created in the suspended state, this field is not set until the first time it is resumed. This field is reset every time a Job is resumed from suspension. It is represented in RFC3339 form and is in UTC.
+
+  <a name="Time"></a>
+  *Time is a wrapper around time.Time which supports correct marshaling to YAML and JSON.  Wrappers are provided for many of the factory methods that the time package offers.*
+-->
+- **startTime** (Time)
+
+  表示作业控制器开始处理作业的时间。在挂起状态下创建 Job 时，直到第一次恢复时才会设置此字段。
+  每次从暂停中恢复作业时都会重置此字段。它表示为 RFC3339 格式的 UTC 时间。
+
+  <a name="Time"></a>
+  **Time 是 time.Time 的包装器，支持正确编码为 YAML 和 JSON。time 包提供的许多工厂方法都提供了包装器。**
+
+<!--
+- **completionTime** (Time)
+
+  Represents time when the job was completed. It is not guaranteed to be set in happens-before order across separate operations. It is represented in RFC3339 form and is in UTC. The completion time is only set when the job finishes successfully.
+
+  <a name="Time"></a>
+  *Time is a wrapper around time.Time which supports correct marshaling to YAML and JSON.  Wrappers are provided for many of the factory methods that the time package offers.*
+-->
+- **completionTime** (Time)
+
+  表示作业完成的时间。不能保证对多个独立操作按发生的先后顺序设置。此字段表示为 RFC3339 格式的 UTC 时间。
+  仅当作业成功完成时才设置完成时间。
+
+  <a name="Time"></a>
```

Comment:
The preview is not good for this line:

https://deploy-preview-34746--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/kubernetes-api/workload-resources/job-v1/

Reply:
This line looks good now, but can be safely dropped.